site/en/blog/renderingng/index.md

```diff
@@ -0,0 +1,569 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'RenderingNG'
+description: >
+  In 2021, we will largely complete the process of designing, building and shipping RenderingNG, a next-generation rendering architecture for Chromium.
+subhead: >
+  Ready for the next generation of web content
+date: 2021-06-15
+authors:
+  - chrishtr
+hero: 'image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/Hdfm4k1cSwfJpffgx3eH.jpg'
+alt: >
+  A pile of different devices.
+---
+
+I'm Chris Harrelson,
+the engineering lead for Rendering (HTML & CSS -> pixels) in Blink.
+I've been deep in the trenches of rendering performance on the web for over eight years,
+with a personal goal of doing whatever I can to make delivering excellent UX on the web faster,
+more reliable, and easier.
+In this blog post and many more to come,
+I'm excited for us to tell you about what we've done in that time to build a new, cutting-edge Chromium rendering engine architecture.
+To achieve this has been an enormous labor of love,
+and I hope you enjoy hearing about it!
+
+In 2021, we will largely complete the process of designing, building and shipping this architecture. Let's call it RenderingNG,
+since it is truly a next-generation rendering architecture that greatly outperforms what came before.
+RenderingNG has been in progress for almost 8 years,
+and represents the collective work of many dedicated developers.
+It unlocks a huge amount of potential for the next generation of fast, fluid, reliable,
+responsive and interactive web content.
+It's also a baseline that we hope defines a new minimum standard for all web rendering engines that developers can rely on.
+
+{% Img
+src="image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/m1d1iB6Ts2HqWRAx8cX5.jpg",
+alt="Sketch of the different elements of RenderingNG",
+width="800", height="789" %}
+
+This blog post is the first in a series,
+where we'll explain what we built, why we built it, and how it works.
+
+In this first post, I'll start with:
+
+- Our North Star goal.
+- The pyramid of success:
+principles that guide our work, and examples of those principles in practice.
+- The features and capabilities that are now possible.
+- A high-level overview of the major project components of RenderingNG.
+
+## North star
+
+The north star goal motivating RenderingNG is that the browser engine implementation,
+and the richness of its rendering APIs, should not be a limiting factor of  UX on the web.
+
+You should not need to worry about browser bugs making features unreliable,
+or breaking your site's rendering.
+
+There should be no mysterious performance cliffs.
+
+You should not need to work around missing built-in features.
```

Comment:
combine this and previous sentences into one paragraph? the "just works" one can be by itself perhaps.